### PR TITLE
Hw/dual camera mount

### PR DESCRIPTION
**Pull request by khrdrgb96**
Made stereo camera mount.
The number of cameras(1 or 2) and the distance between camera are parameterized.

### CODE REVIEW

File: hardware/src/CamRobot/RaspiMount_protoboard.scad [hunk 1]
```diff
@@ -79,18 +78,46 @@ module side_stopper(){
...
+        translate([0, 0.2, 0.2])
+            rotate([90, 0, 0])
+            minkowski()
+            {
+                cube([(camera_hole_pitch_w )/2+0.1, camera_hole_pitch_h+upper_hole_h , 0.1]);
+                cylinder(r=0.2,h=0.1);
+            }
+        
+        //translate([0, 0, 0.2])
+           // cube([(camera_hole_pitch_w+0.5)/2, 0.2,camera_hole_pitch_h+upper_hole_h+0.1]);
```
Comment (poifull10) [inline hardware/src/CamRobot/RaspiMount_protoboard.scad]: please remove useless codes
```suggestion
```

File: hardware/src/CamRobot/RaspiMount_protoboard.scad [hunk 2]
```diff
@@ -1,7 +1,22 @@
...
 
-module cm_half(camera_hole_r, camera_hole_pitch_w, camera_hole_pitch_h, upper_hole_h){
+//args
+// num_camera: number of camera(1 or 2)
+// camera_hole_r: radius of camera hole
+// camera _hole_pitch_w: horizontal pitch of camera holes
+// camera_hole_pitch_h: vertical pitch of camera holes
+// camera_distance: distance between hole of 2 cameras. It is used  only for stereo camera mount
+// upper_hole_h: height of upper camera holes
+module RaspiCameraMount(num_camera=2, camera_hole_r=0.1, camera_hole_pitch_w=2.1, camera_hole_pitch_h=1.25, camera_distance=1,  upper_hole_h=1){
```
Comment (poifull10) [inline hardware/src/CamRobot/RaspiMount_protoboard.scad]: This module does not support multiple cameras more than three, so please replace num_cameras with a boolean argument as follows:  
```suggestion
module RaspiCameraMount(stereo_camera=false, camera_hole_r=0.1, camera_hole_pitch_w=2.1, camera_hole_pitch_h=1.25, camera_distance=1,  upper_hole_h=1){
```

Comment (poifull10) [reply hardware/src/CamRobot/RaspiMount_protoboard.scad]: Please fix all modules as well.

**Review by poifull10 — CHANGES_REQUESTED** [review]

File: hardware/src/CamRobot/RaspiMount_protoboard.scad (hunk 2 above)
Comment (khrdrgb96) [reply hardware/src/CamRobot/RaspiMount_protoboard.scad]: DONE

**Review by poifull10 — APPROVED** [review]
Great work!

**Pull request merged**